### PR TITLE
Armonia Meta Chain - Eva and Wall-e for EVM

### PR DESCRIPTION
**Pull request by masteraychen**
Need to have unique chain IDs reserved for our EVM-compatible mainnet & testnet chains:
 - Eva chain: mainnet
 - Wall-e chain: testnet

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Comment by ligi** [comment]
also https://evascan.io is not reachable

**Comment by masteraychen** [comment]
removed the mainnet explorer as it's being developed.

**Comment by masteraychen** [comment]
Pls kindly let me know if anything that needs be done further

File: _data/chains/eip155-160.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Armonia Eva Chain Mainnet",
+  "chain": "Eva",
+  "rpc": ["https://evascan.io/api/eth-rpc/"],
+  "faucets": [""],
```
Comment (ligi) [inline _data/chains/eip155-160.json:5]:
```suggestion
  "faucets": [],
```

File: _data/chains/eip155-161.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Armonia Eva Chain Testnet",
+  "chain": "Wall-e",
+  "rpc": ["https://testnet.evascan.io/api/eth-rpc/"],
+  "faucets": [""],
```
Comment (ligi) [inline _data/chains/eip155-161.json:5]:
```suggestion
  "faucets": [],
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by masteraychen** [comment]
Pls revive it

**Pull request merged**